### PR TITLE
Updated REST API doc for v2.8 release

### PR DESCRIPTION
**Pull request by KonikaChaurasiya-GSLab**
Updated REST API doc for v2.8 release

### CODE REVIEW

**Comment by manuwelakanade-GSLAB** [comment]
Hi @JosephPJordan : Can you please review and merge this pull request. This is an additional change in the REST API docs which we missed in the previous pull request. Thank you.

File: docs/rest_api.rst [hunk 1]
```diff
@@ -1945,6 +1945,1154 @@ getasynctaskresponse.json
...
+|                    |                                                                                                            |
+|                    | ]                                                                                                          |
+|                    |                                                                                                            |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Error Response     | Code: 400                                                                                                  |
+|                    |                                                                                                            |
+|                    | Content: {error: Bad request}                                                                              |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Notes              | This is added in the postman collection.                                                                   |
```
Comment (JosephPJordan) [inline docs/rest_api.rst:2007]: Postman 
(cap P)

File: docs/rest_api.rst [hunk 2]
```diff
@@ -1945,6 +1945,1154 @@ getasynctaskresponse.json
...
+|                    |                                                                                                            |
+|                    | }                                                                                                          |
+|                    |                                                                                                            |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Error Response     | Code: 400                                                                                                  |
+|                    |                                                                                                            |
+|                    | Content: {error: Bad request}                                                                              |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Notes              | This is added in the postman collection.                                                                   |
```
Comment (JosephPJordan) [inline docs/rest_api.rst:2393]: Postman

File: docs/rest_api.rst [hunk 3]
```diff
@@ -1945,6 +1945,1154 @@ getasynctaskresponse.json
...
+|                    |                                                                                                            |
+|                    | }                                                                                                          |
+|                    |                                                                                                            |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Error Response     | Code: 400                                                                                                  |
+|                    |                                                                                                            |
+|                    | Content: {error: Bad request}                                                                              |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Notes              | This is added in the postman collection .                                                                  |
```
Comment (JosephPJordan) [inline docs/rest_api.rst:2171]: Postman collection.

File: docs/rest_api.rst [hunk 4]
```diff
@@ -1945,6 +1945,1154 @@ getasynctaskresponse.json
...
++--------------------+------------------------------------------------------------------------------------------------------------+
+
+updatefastdata.json
+````````````````````
+
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Title              | create/update FAST applications/partitions                                                                 |
```
Comment (JosephPJordan) [inline docs/rest_api.rst:2178]: Create

File: docs/rest_api.rst [hunk 5]
```diff
@@ -1945,6 +1945,1154 @@ getasynctaskresponse.json
...
+|                    | Content: {error: Bad request}                                                                              |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Notes              | This is added in the postman collection.                                                                   |
++--------------------+------------------------------------------------------------------------------------------------------------+
+
+updatefastdata.json
+````````````````````
+
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Title              | create/update FAST applications/partitions service-discovery template to support EP attach detach feature  |
```
Comment (JosephPJordan) [inline docs/rest_api.rst:2400]: Create

File: docs/rest_api.rst [hunk 6]
```diff
@@ -1945,6 +1945,1154 @@ getasynctaskresponse.json
...
+|                    | }                                                                                                          |
+|                    |                                                                                                            |
+|                    | }                                                                                                          |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Error Response     | Code: 400                                                                                                  |
+|                    |                                                                                                            |
+|                    | Content: {error: Bad request}                                                                              |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Notes              | This is added in the postman collection.                                                                   |
```
Comment (JosephPJordan) [inline docs/rest_api.rst:2681]: Postman

File: docs/rest_api.rst [hunk 7]
```diff
@@ -1945,6 +1945,1154 @@ getasynctaskresponse.json
...
+|                    |                                                                                                            |
+|                    | }                                                                                                          |
+|                    |                                                                                                            |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Error Response     | Code: 400                                                                                                  |
+|                    |                                                                                                            |
+|                    | Content: {error: Bad request}                                                                              |
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Notes              | This is added in the postman collection.                                                                   |
```
Comment (JosephPJordan) [inline docs/rest_api.rst:2803]: Postman

File: docs/rest_api.rst [hunk 8]
```diff
@@ -1945,6 +1945,1154 @@ getasynctaskresponse.json
...
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Notes              |                                                                                                            |
++--------------------+------------------------------------------------------------------------------------------------------------+
+
+
+searchendpoint.json
+`````````````````````
+
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Title              | Searches Endpoint across login BIG-IPs                                                                     |
```
Comment (JosephPJordan) [inline docs/rest_api.rst:2955]: (not exactly sure what this means, but maybe:)
Searches Endpoints across logged in BIG-IPs
(maybe login BIG-IPs means something else, if so, leave it)

File: docs/rest_api.rst [hunk 9]
```diff
@@ -1945,6 +1945,1154 @@ getasynctaskresponse.json
...
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Notes              |                                                                                                            |
++--------------------+------------------------------------------------------------------------------------------------------------+
+
+
+searchendpoint.json
+`````````````````````
+
++--------------------+------------------------------------------------------------------------------------------------------------+
+| Title              | Searches Endpoint for specified BIG-IP                                                                     |
```
Comment (JosephPJordan) [inline docs/rest_api.rst:3028]: Searches Endpoints (or 'the Endpoint') for the specified BIG-IP

**Review by JosephPJordan — CHANGES_REQUESTED** [review]
A few minor changes

**Comment by manuwelakanade-GSLAB** [comment]
Hi @JosephPJordan : Thanks for the review. I've addressed all the comments and made changes accordingly. Can you please merge the pull request. Thank you.

**Review by JosephPJordan — APPROVED** [review]

**Pull request merged**